label_studio/tests/sdk/test_ml.py

```diff
@@ -0,0 +1,198 @@
+import pytest
+from label_studio_sdk.client import LabelStudio
+
+from label_studio.tests.utils import register_ml_backend_mock
+
+
+@pytest.fixture
+def ml_backend_for_test_batch_predictions(ml_backend):
```

Comment:
Is this any different to the pytest fixture defined in test_predict.py? If not, I think it'd be better to have one fixture for both test suites to use.